src/engraving/rw/write/twrite.cpp

```diff
@@ -1604,6 +1604,12 @@ void TWrite::write(const Hairpin* item, XmlWriter& xml, WriteContext& ctx)
     writeProperty(item, xml, Pid::SNAP_BEFORE);
     writeProperty(item, xml, Pid::SNAP_AFTER);
 
+    for (const StyledProperty& spp : *item->styledProperties()) {
```

Comment:
Do we need to prevent overlap with `TextLineBase::textLineBasePropertyIds()` (see `writeProperties(TextLineBase*)`)? Or do they not overlap anyway?